src/sass/_utils.scss

```diff
@@ -31,3 +31,25 @@
 }
 
 // sass-lint:enable no-important
+
+@keyframes sgAnimationFadeIn {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+.sg-animation-fade-in-slow {
```

Comment:
what about fadeOut? fadeIn gonna miss it 🤗

Reply:
@matzimowski that's true :) please keep in mind this is kind of temporary. To cover fade out properly we need transition component taking care of keeping instance until animation ends. I hope we get there when we will have motion designer :)